src/utils/ipfs.js

```diff
@@ -0,0 +1,14 @@
+const DAGNode = require('ipld-dag-pb').DAGNode;
```

Comment:
I wonder if to store/get/resolve reviews can go via the api/client/server?

I am thinking only for the crawled reviews, or to say, during airdrop case.

That way, the wallets during airdrops, don't have to start an ipfs instance? Does it make sense?

Reply:
They have to start it for publishing. But for reading, they can use the API. We can figure out which to use later, they both work.

Reply:
Remember that this 'ipld-dag-pb' module does not require IPFS to be running at all! It's just to serialize the data and calculate the multihash